### PR TITLE
fix initialization of Artemis Eigen volume directions

### PR DESCRIPTION
**Pull request by fedorov**

### CODE REVIEW

File: DicomUltrasoundPlugin/DicomUltrasoundPlugin.py [hunk 1]
```diff
@@ -701,6 +701,12 @@ def loadEigenArtemis3DUS(self,loadable):
     volumesLogic = slicer.modules.volumes.logic()
     outputVolume = volumesLogic.AddArchetypeScalarVolume(filePath,name,0,fileList)
 
+    ijk2ras = vtk.vtkMatrix4x4()
+    outputVolume.GetIJKToRASDirectionMatrix(ijk2ras)
```
Comment (lassoan) [inline DicomUltrasoundPlugin/DicomUltrasoundPlugin.py]: Are you sure you want to hardcode two values? If it is a LPS/RAS conversion then you should multiply the ijk2ras matrix by `diag(-1,-1,1,1)` matrix. If you want to keep this hardcoded setting of these two values then at least add a comment (why it is needed and why exactly those two values need to be changed like this).

**Comment by fedorov** [comment]
@lassoan I am still working on this with the submitter of the data to make sure the orientation is correct. I will remove WIP from the issue title and ping you when I believe it is ready for integration.

**Comment by fedorov** [comment]
@lassoan this should be ready to go - can we proceed with review and merge?

**Comment by lassoan** [comment]
Thank you, merged. Setting origin and spacing then getting IJKToRAS and then applying rotations and setting again IJKToRAS is not elegant, but it's OK. Instead, it would have been nicer to create a vtkTransform, call Scale, Translate, Rotate, etc. methods on it and then set the resulting matrix in IJKToRAS.

**Comment by fedorov** [comment]
Good point Andras. I am using VTK once in a blue moon, so I will be the first to admit I am not equipped to produce elegant solutions :-\

**Pull request merged**